bigbluebutton-html5/public/locales/en.json

```diff
@@ -300,12 +300,11 @@
     "app.presentationUploader.exportCurrentStatePresentation": "Send out a download link for the presentation including whiteboard annotations",
     "app.presentationUploader.enableOriginalPresentationDownload": "Enable download of the presentation ({0})",
     "app.presentationUploader.disableOriginalPresentationDownload": "Disable download of the original presentation ({0})",
+    "app.presentationUploader.withWhiteboardAnnotations": " (with whiteboard annotations)",
```

Comment:
I recommend sticking with the existing key `app.presentationUploader.export.withWhiteboardAnnotations`, as it has already been translated by the community.

Additionally, it's more efficient to leave decisions about white-space and parentheses to the UI design, focusing solely on translating the text. This way, we maintain flexibility in our UI design.